backend/tests/unit/models/test_websocket_models.py

```
@@ -21,6 +21,7 @@
     SystemHealthUpdate,
     WebSocketMessage,
 )
+from tests.utils import ModelValidationTester, TestUtils
```

Ruff (0.12.2)

`tests.utils.ModelValidationTester` imported but unused

Remove unused import

(F401)